jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryDetails.kt

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.temporal.ChronoUnit
+import java.util.Date
+
+typealias SelectedLogGroups = MutableList<String>
+
+data class QueryDetails(
+    val logGroups: SelectedLogGroups,
+    val timeRange: TimeRange,
+    val query: QueryString
```

Comment:
Do we need to get `query` anywhere vs `getQueryString`? Can we make `query` private?

Reply:
`query` is used when populating the dialog from an existing `QueryDetails`